kumascript/macros/CSS_Ref.ejs

```diff
@@ -9,7 +9,7 @@
   $1 - Item grouping type
        (valid values: 'alphabetically', 'no'; defaults to 'alphabetically')
   $2 - Array of strings used to filter the items by their status
-       (valid values: 'standard', 'experimental', "nonstandard', 'obsolete';
+       (valid values: 'standard', 'experimental', "nonstandard';
```

Comment:
This is not driven by a tag but by arguments given to the macro. **However** this macro is only used in one page (https://developer.mozilla.org/en-US/docs/Web/CSS/Reference#index), and in that case it's not passed any arguments, so this change is OK.